tests/introspection/iOS/iOSApiProtocolTest.cs

```diff
@@ -637,6 +637,16 @@ protected override bool Skip (Type type, string protocolName)
 					return !TestRuntime.CheckXcodeVersion (8, 0);
 				}
 				break;
+
+			case "UISpringLoadedInteractionSupporting": // new conformations of UISpringLoadedInteractionSupporting in iOS 11
+				switch (type.Name) {
+				case "UIButton": // UIButton (SpringLoading) category
+				case "UIAlertController": // UIAlertController (SpringLoading) category
+				case "PKPaymentButton": // UIButton (SpringLoading) category
+				case "PKAddPassButton": // UIButton (SpringLoading) category
+					return true;
```

Comment:
does it work on iOS 11 ?
Normally this would be `return !TestRuntime.CheckXcodeVersion (9, 0);`, but the above is unclear. i.e. if it fails on iOS11 then it's something we should file a rdar for

Also since its' iOS only the whole `case` could be inside `#if IOS`

Reply:
Yup I got this wrong, thought it failed because the conformance was defined in categories. Filing a radar now as my Swift test case for protocol conformance doesn't work (;

Reply:
https://trello.com/b/ZXs89x7A